Tools/flatpak/flatpakutils.py

```diff
@@ -955,15 +947,13 @@ def envvar_in_suffixes_to_keep(envvar):
                 # Spawn the sccache server in background, and avoid recursing here, using a bool keyword.
                 _log.debug("Pre-starting the SCCache dist server")
                 self.run_in_sandbox("sccache", "--start-server", env=sccache_environment, building_local_deps=building_local_deps,
-                                    extra_flatpak_args=[share_network_option], start_sccache=False)
+                                    start_sccache=False)
 
             # Forward sccache server env vars to sccache clients.
             sandbox_environment.update(sccache_environment)
 
         if self.use_icecream and not skip_icc:
             _log.debug('Enabling the icecream compiler')
-            flatpak_command.extend([share_network_option, "--filesystem=home"])
```

Comment:
Why are you removing the `--filesystem=home` from here?

Reply:
It's useless because `--filesystem=host` is added already.

Reply:
I see.